sdk/python/jobs/automl-standalone-jobs/automl-forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1429 @@
+    "actual_column_name = \"automl_actuals\"\n",
+    "forecast_origin_column_name = \"automl_forecast_origin\"\n",
+    "\n",
+    "automl_fcst = pd.read_parquet(os.path.join(mm_output_dir, forecast_table_relative_path))\n",
+    "base_fcst = pd.read_parquet(os.path.join(base_output_dir, forecast_table_relative_path))\n",
+    "\n",
+    "merge_columns = time_series_id_column_names + [actual_column_name]\n",
+    "merge_columns.extend([time_column_name, forecast_origin_column_name])\n",
+    "\n",
+    "backtest = automl_fcst.merge(\n",
+    "    base_fcst.rename(columns={forecast_column_name: base_forecast_column_name}),\n",
+    "    on=merge_columns,\n",
+    "    how=\"inner\",\n",
+    ")\n",
+    "\n",
+    "print(f\"AutoML forecast table size: {automl_fcst.shape}\\n---\")\n",
+    "print(f\"Base forecast table size: {base_fcst.shape}\\n---\")\n",
+    "print(f\"Merged forecast table size: {backtest.shape}\\n---\")\n",
+    "backtest.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scripts.helper_scripts import draw_one_plot\n",
+    "from matplotlib import pyplot as plt\n",
+    "from matplotlib.backends.backend_pdf import PdfPages\n",
+    "\n",
+    "plot_filename = \"forecast_vs_actual.pdf\"\n",
+    "\n",
+    "pdf = PdfPages(os.path.join(os.getcwd(), \"./output\", plot_filename))\n",
+    "for _, one_forecast in backtest.groupby(time_series_id_column_names):\n",
+    "    one_forecast[time_column_name] = pd.to_datetime(one_forecast[time_column_name])\n",
+    "    one_forecast.sort_values(time_column_name, inplace=True)\n",
+    "    draw_one_plot(\n",
+    "        one_forecast,\n",
+    "        time_column_name,\n",
+    "        target_column_name,\n",
+    "        time_series_id_column_names,\n",
+    "        [actual_column_name, forecast_column_name, base_forecast_column_name],\n",
+    "        pdf,\n",
+    "        plot_predictions=True,\n",
+    "    )\n",
+    "pdf.close()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from IPython.display import IFrame\n",
+    "\n",
+    "IFrame(os.path.join(\"./output/forecast_vs_actual.pdf\"), width=800, height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 10. Deployment\n",
+    "\n",
+    "In this section, we will illustrate how to deploy and inference models using batch endpoint. Batch endpoints are endpoints that are used to do batch inferencing on large volumes of data over in asynchronous way. Batch endpoints receive pointers to data and run jobs asynchronously to process the data in parallel on compute clusters and store outputs to a datastore for further analysis. For more information on batch endpoints see this [link](https://learn.microsoft.com/en-us/azure/machine-learning/concept-endpoints-batch?view=azureml-api-2).\n",
```

Comment:
"Batch endpoints are endpoints that are used to do batch inferencing on large volumes of data over in asynchronous way."

You may want to remove "over"?

Reply:
Done.